.vib/fluentd/goss/fluentd.yaml

```diff
@@ -0,0 +1,10 @@
+command:
+  check-no-pem-files:
+    exec: find /opt/bitnami/fluentd -name "*.pem" | grep .
+    exit-status: 1
+  check-fluent-plugin-detect-exceptions:
+    exec: ls -l /opt/bitnami/fluentd/gems/fluent-plugin-detect-exceptions-*/lib/fluent/plugin/*.rb 2>/dev/null | grep ".*-rwxrwxr-x"
+    exit-status: 0
```

Comment:
This test might end up producing false positives: if there are multiple files but only one of them matches the permissions, it will still succeed.

Reply:
How would you recommend transforming the test so that these false positives don't occur?

Reply:
Maybe a good approach would be checking the opposite case: if one of the files does not match, then produce an error. But it just an idea :)